### PR TITLE
[ZEPPELIN-2003] Remove PostgresqlInterpreter

### PR DESCRIPTION
**Pull request by jongyoul**
### What is this PR for?
We don't have to maintain PostgresqlInterpreter because JDBCInterpreter covers all functions of PostgresqlInterpreter. It reduces maintenance costs.

### What type of PR is it?
[Feature]

### Todos
* [x] - Remove files and lines related to PostgresqlInterpreter

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-2003

### How should this be tested?
N/A

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? Yes
* Does this needs documentation? Yes, but already documented in jdbc docs

### CODE REVIEW

**Comment by felixcheung** [comment]
shall we leave the postgreSQL interpreter md file to point to JDBC one? I think our users might be looking for it

**Comment by jongyoul** [comment]
@felixcheung Sounds like a good idea!

**Comment by prabhjyotsingh** [comment]
LGTM!

**Comment by jongyoul** [comment]
I made a link. @AhyoungRyu Can you verify the doc?

File: docs/interpreter/postgresql.md [hunk 1]
```diff
@@ -1,230 +1,28 @@
...
+-->		
+{% include JB/setup %}		
+		
+# PostgreSQL, Apache HAWQ (incubating) Interpreter for Apache Zeppelin		
+		
+<div id="toc"></div>		
+		
+## Important Notice		
+		
+Postgresql Interpreter is deprecated and merged into [JDBC Interpreter](./jdbc.md).
```
Comment (AhyoungRyu) [inline docs/interpreter/postgresql.md]: The link should be `./jdbc.html`. And how about pointing Postgresql 's setting example in JDBC interpreter page? e.g. 

```Postgresql interpreter is deprecated and merged into [JDBC Interpreter](./jdbc.html). You can use it with JDBC Interpreter as same functionality. See [Postgresql setting example](./jdbc.html#postgres) for more detailed information.```

Comment (jongyoul) [inline docs/interpreter/postgresql.md]: Thanks!!

**Comment by jongyoul** [comment]
@AhyoungRyu Please check the change.

**Comment by AhyoungRyu** [comment]
@jongyoul Looks good!

**Comment by felixcheung** [comment]
LGTM

**Pull request closed without merging**